File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q08.js
import { question } from "readline-sync";
//Input 
    const num1 = parseInt(question("Enter a number value: "))
    const num2 =  parseInt(question("Enter a second number value: "))

//Processing
    const sum = num1 + num2
    const subtraction = num1 - num2 
    const result = sum / subtraction

//Output
    console.log(`The result of the sum of {num1} and {num2} by subtraction of ${num1} and ${num2} is: ${result} `)

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q08.py
def main():
#Input
    num1 = int(input("Enter a number value: "))
    num2 = int(input("Enter a second number value: "))

#Processing
    sum = num1 + num2
    subtraction = num1 - num2
    result = sum/subtraction

#Output
    print(f"The result of the sum of the number {num1} and {num2} by the subtraction of {num1} and {num2} is equal to: {result}")

main()